### PR TITLE
Adding updated readme

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

**Comment by secondsun** [comment]
@danielpassos @corinnekrych quick review?

File: README.md [hunk 1]
```diff
@@ -1 +1,122 @@
...
+
+If you do not have access to a RHMAP instance, you can sign up for a free instance at [https://openshift.feedhenry.com/](https://openshift.feedhenry.com/).
+
+## How do I run it?  
+
+### RHMAP Studio
+
+You can import this project as a client app inside of RHMAP Studio.  First you will need to create a project based on the  "SAML Example Project" template.  In this project you will add a new client app by importing an existing app.  Select "Native Android" as your app type and import from this git repository. 
+
+### Build instructions for Open Source Development
```
Comment (corinnekrych) [inline README.md]: ### Local Clone (ideal for Open Source Development)

@secondsun ^^ this is more the title you used in other templates, let 's be consistent

Comment (secondsun) [inline README.md]: Good catch!

File: README.md [hunk 2]
```diff
@@ -1 +1,122 @@
...
+Summary: A demonstration of how to integrate with the [FeedHenry SAML PoC](https://github.com/feedhenry-templates/saml-cloud-app)  
+Community Project : [Feed Henry](http://feedhenry.org/)  
+Target Product: RedHat Mobile Application Platform aka RHMAP    
+Product Versions: RHMAP 3.8.0+   
+Source: https://github.com/feedhenry-templates/pushstarter-android-app  
+Prerequisites: fh-android-sdk : 3.0.+, Android Studio : 1.4.0 or newer, Android SDK : 22+ or newer
+
+## What is it?
+
+This application will subscribe to a push service running in a RHMAP instance. The user can send messages to the device using RHMAP and view them on the device.
```
Comment (danielpassos) [inline README.md:15]: push service?

Comment (secondsun) [inline README.md:15]: Couldn't think of a better wording.

Summers Pittman

> > Phone:404 941 4698
> > Java is my crack.

**Pull request merged**